osquery/tables/system/windows/groups.cpp

```diff
@@ -0,0 +1,142 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#define _WIN32_DCOM
+#define WIN32_LEAN_AND_MEAN
+#include <Windows.h>
+// clang-format off
+#include <LM.h>
+// clang-format on
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/logger.h>
+
+#include "osquery/core/process.h"
+#include "osquery/core/windows/wmi.h"
+#include "osquery/core/windows/process_ops.h"
+#include "osquery/tables/system/windows/registry.h"
+#include "osquery/core/conversions.h"
+
+namespace osquery {
+
+// Get the relative identifier (RID) from a security identifier (SID):
+unsigned long getRidFromSid(PSID sidPtr) {
+  BYTE* countPtr = GetSidSubAuthorityCount(sidPtr);
+  unsigned long indexOfRid = static_cast<unsigned long>(*countPtr - 1);
+  unsigned long* ridPtr = GetSidSubAuthority(sidPtr, indexOfRid);
+  return *ridPtr;
+}
+
+namespace tables {
+
+std::unique_ptr<BYTE[]> GetSid(LPCWSTR accountName) {

```

Comment:
Any reason for the capital `GetSid` here? Could this be changed to `getSid` to match the other function prototypes?

Reply:
There is no reason other than I had no preference, and was probably influenced by all of the Windows APIs I was using. I will change it to be consistent with the others as suggested.